doc/source/dev/contributor/quickstart_ubuntu.rst

```diff
@@ -30,11 +30,9 @@ development environment with Docker may be more reliable.*
 Building SciPy
 --------------
 
-*Consider watching the companion videos* `Anaconda SciPy Dev: Part I (macOS)`_ and `Anaconda SciPy Dev: Part II (macOS)`_ *before starting. Although these videos are intended for macOS users, some of the procedures are the same or very similar to those described here.*
-
 #. Download, install, and test the latest release of the `Anaconda Distribution of Python`_. In addition to the latest version of Python 3, the Anaconda distribution includes dozens of the most popular Python packages for scientific computing, the Spyder integrated development environment (IDE), the ``conda`` package manager, and tools for managing virtual environments.
 
-   `In a terminal, navigate <https://help.ubuntu.com/community/UsingTheTerminal>`_ to the location in which you'd like to install Anaconda. You can download the file using the terminal command ``curl -O URL_OF_FILE``, where ``URL_OF_FILE`` is to be replaced with the URL of the Anaconda installer ``.sh`` file found at the Anaconda Distribution website. Run the installer by entering ``bash file.sh``, where ``file.sh`` is again to be replaced with the full name of the downloaded file. This starts the installation process. From there, simply follow the prompts, including the "Next Steps" at the end after the installer finishes.
+   `In a terminal, navigate <https://help.ubuntu.com/community/UsingTheTerminal>`_ to the location in which you'd like to install Anaconda. You can download the file using the terminal command ``curl -O URL_OF_FILE``, where ``URL_OF_FILE`` is to be replaced with the URL of the Anaconda installer ``.sh`` file found at the Anaconda Distribution website. Run the installer by entering ``bash file.sh``, where ``file.sh`` is again to be replaced with the full name of the downloaded file. This starts the installation process. From there, simply follow the prompts, including the "Next Steps" at the end after the installer finishes. You might also need to restart your terminal window or enter ``source ~/.bashrc`` for all the changes to take effect.
```

Comment:
Not strictly related to this PR, but IMO you should either give the user a complete curl command to download `conda` just in terminal, or give them a link to the download page on the anaconda website. Giving them a link to follow in their browser to find the correct URL to pass to curl just seems confusing.

Reply:
I think the filename changes as new versions are released, so I don't think we can provide the complete command. 

Yes, in many cases, the `curl` command would be unnecessary. In many cases, if they can browse to the website, they can also download in their browser and use a graphical installer. I included the `curl` command for WSL and remote terminal users who can browse to the site using their native OS but can only interact via terminal with the machine on which they're doing the installation. That's what I do.

Reply:
I would be in favour of leaving out curl then. I think it's reasonable to assume that anyone using a remote terminal knows how to transfer a file from their local machine.

Reply:
I didn't, and that's why I included it. But I'll take it out.